package.json

```diff
@@ -22,9 +28,11 @@
     "react-bootstrap": "1.0.0-beta.5",
     "react-dom": "16.8.4",
     "react-scripts": "2.1.3",
-    "truffle": "5.1.39",
-    "truffle-hdwallet-provider": "^1.0.17",
     "web3": "^1.2.11",
```

Comment:
Can we drop `web3` if we're adding `ethers`?

Reply:
of course :) I actually had meant to ask you if you had any strong feelings about this?
hardhat uses ethers by default, and I personally find it more confortable to use. but since I imagine you're already using web3 on the rest of front-end as well, maybe you want to stick with it?

Reply:
No strong feelings from my part, it is indeed what I'm using on the mvp but if it is what ships by default with hardhat and you are comfortable with it we might as well stick with it. @andreasvilela what do you think?